### PR TITLE
[cker] Bring SGD kernel from tensorflow v2.12.1

### PR DESCRIPTION
**Pull request by ragmani**
This commit brings Gradient Descent kernel from tensorflow v2.12.1.

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

**Review by glistening — APPROVED** [review]
LGTM

**Comment by hseok-oh** [comment]
@ragmani Please resolve conflict.

**Comment by ragmani** [comment]
Resolved conflict.

**Review by glistening — APPROVED** [review]
LGTM

**Comment by ragmani** [comment]
@hseok-oh  PTAL

**Comment by ragmani** [comment]
@Samsung/one_onert PTAL

**Review by jyoungyun — APPROVED** [review]
LGTM
